chat-formatter/src/com/eternalcode/formatter/ChatFormatterPlugin.java

```diff
@@ -66,7 +70,8 @@ public void onEnable() {
         new Metrics(this, 15199);
 
         Stream.of(
```

Comment:
Tu można wsm PandaStreama użyć bo tam chyba lepszy jest, ale niewymagane

Reply:
PandaStream to wrapper do Java Stream API. Po prostu rozszerza jego działanie.